python file>.py

```diff
@@ -0,0 +1,64 @@
+fs_pairs = {'bob': 'bobby',
+            'apple': 'tree',
+            'dog': 'wolf'}
+choice = None
+def choices():
+    print('''
+
+           Your choices right now:
+
+           1: Exit
+           2: Add son father pair
+           3: Replace a son's father with another 
+           4: Delete son father pair
+           5: View son father pairs
+
+           ''')
+while choice != '1':
+    choices()
+    choice = input('What is your choice? : ')
+
+    while choice != '1' or '2' or '3' or '4' or '5':
```

Comment:
You definitely do not need two `while` loops.

Reply:
how do i make it so that it prints "try again" while choice isn't 1, 2, 3, 4, or 5 then

Reply:
Let's just remove the `"Try Again"` for now